lms/envs/common.py

```diff
@@ -848,6 +848,15 @@
 ##### Feedback submission mechanism #####
 FEEDBACK_SUBMISSION_EMAIL = None
 
+#### Help Modal Links ####
+if FEATURES.get('ENABLE_FEEDBACK_SUBMISSION'):
```

Comment:
I don't think we even need this; we should always initialize this to an empty list. 

Templates may chose to not do anything with the values, but at least they can access the field without an error.